src/component/Favorites/Favorites.component.jsx

```diff
@@ -6,7 +6,7 @@ const FavoritesView = ({ favorites, setFavorites }) => {
   const navigate = useNavigate();
 
   const handleAnimalClick = (animalName) => {
-    navigate(`/game?animal_name=${animalName}`);
+    navigate(`/game?animal_id=${animalName}`);
```

Comment:
This is a great change, animal id is better than animal name for clarity and getting back precisely the right animal.